scripts/travis-emscripten/build_emscripten.sh

```diff
@@ -62,9 +62,8 @@ cd $WORKSPACE
 mkdir -p build
 cd build
 cmake \
-  -DCMAKE_TOOLCHAIN_FILE=$EMSCRIPTEN/cmake/Modules/Platform/Emscripten.cmake \
+  -DCMAKE_TOOLCHAIN_FILE=../cmake/toolchains/emscripten.cmake \
   -DCMAKE_BUILD_TYPE=Release \
-  -DEMSCRIPTEN=1 \
```

Comment:
If this is not set anymore, will if the `EthCompilerSettings` still work?

Reply:
This is set by the emscripten toolchain file from Emscripten SDK. So it not works like `if(APPLE)` or `if(MSVC)`.